ch.protonmail.protonmail-bridge.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: ch.protonmail.protonmail-bridge
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "18.08"
+command: protonmail-bridge
+rename-desktop-file: ProtonMail_Bridge.desktop
+rename-icon: ProtonMail_Bridge
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Notifications
+modules:
+
+  - name: libsecret
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/libsecret/-/archive/0.18.8/libsecret-0.18.8.tar.gz
+        sha256: 33ee5dfd3556931b81d47111890c8b9c51093b4ced18e0e87f51c1769e24d43c
+    config-opts:
+      - --disable-manpages
+      - --disable-gtk-doc
+      - --disable-static
+      - --disable-introspection
+    cleanup:
+      - /bin
+      - /include
+      - /lib/*.la
+      - /lib/pkgconfig
+
+  - name: protonmail-bridge
+    buildsystem: simple
+    build-commands:
+      - ar x protonmail-bridge_1.1.6-1_amd64.deb
+      - rm -f protonmail-bridge_1.1.6-1_amd64.deb
+      - tar -xf data.tar.xz
+      - rm -f data.tar.xz
+      - mv usr/lib/* /app/lib/
+      - mv usr/share/* /app/share/
+      - rm -rf usr
+      - ln -s /app/lib/protonmail/bridge/protonmail-bridge /app/bin/protonmail-bridge
+      - install -Dp -m 644 /app/share/icons/protonmail/ProtonMail_Bridge.svg /app/share/icons/hicolor/scalable/apps/ProtonMail_Bridge.svg
+      - install -Dp -m 644 ch.protonmail.protonmail-bridge.appdata.xml /app/share/metainfo/ch.protonmail.protonmail-bridge.appdata.xml
+    sources:
+      - type: file
+        url: https://protonmail.com/download/beta/protonmail-bridge_1.1.6-1_amd64.deb
```

Comment:
what's the licence around this file? does it allow redistribution? if not you'll need to use extra-data

Reply:
It comes bundled with a MIT license, I don't know if this makes sense since it is propietary (The code should be released at some point this year).

Reply:
Should I change
```diff
-  <project_license>LicenseRef-proprietary</project_license>
+  <project_license>MIT</project_license>
```
Or this is more involved?

Reply:
No, that's good enough.